files/ini_file.py

```diff
@@ -186,10 +186,10 @@ def do_ini(module, filename, section=None, option=None, value=None, state='prese
                                 else:
                                     index = index + 1
                         break
-                else:
-                    # comment out the existing option line
+                elif state == 'absent':
```

Comment:
So, since the choice for state is either "absent" or "present", am I missing something for explicitely checking for "absent" ?